<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Map Link:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_mapLink">
       <property name="text">
        <string>https://www.openstreetmap.org/#map=11/39.9208/32.8539</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Seperator:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_seperator">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_help">
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_getCoordinates">
       <property name="text">
        <string>Get Coordinates</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_latitude">
         <property name="text">
          <string>Latitude</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_swap">
         <property name="text">
          <string>Swap</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_longitude">
         <property name="text">
          <string>Longitude</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_elevation">
       <property name="text">
        <string>Process Elevation Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Length of the Square Edge (0 - 4 km)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_length">
       <property name="text">
        <string>Length is 0.5 km</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_length">
       <property name="maximum">
        <number>40</number>
       </property>
       <property name="pageStep">
        <number>40</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="topMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_downloadData">
         <property name="text">
          <string>Download Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_showWeb">
         <property name="text">
          <string>Show in Web Browser</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Progress:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_status">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
